staging/cse/windows/azurecnifunc.tests.suites/AzureCNI.Overlay.conflist

```diff
@@ -1,18 +1,19 @@
 {
     "cniVersion": "0.3.0",
     "name": "azure",
-    "adapterName" : "",
+    "adapterName": "",
     "plugins": [
         {
             "type": "azure-vnet",
             "mode": "bridge",
             "bridge": "azure0",
+            "executionMode": "v4swift",
             "capabilities": {
                 "portMappings": true,
                 "dns": true
             },
             "ipam": {
-                "type": "azure-vnet-ipam"
+                "type": "azure-cns"
```

Comment:
not sure if it's necessary here but overlay also has [this](https://github.com/Azure/azure-container-networking/blob/95155ca9b4079cfe9ce4e93c04de419e28a3ad3c/cni/azure-windows-swift-overlay.conflist#L17) in config